parm/config/gefs/config.fcst

```diff
@@ -43,6 +43,7 @@ export FHOUT=${FHOUT_GFS}
 export FHOUT_HF=${FHOUT_HF_GFS}
 export FHOUT_OCN=${FHOUT_OCN_GFS}
 export FHOUT_ICE=${FHOUT_ICE_GFS}
+export FHZER=@FHZER@
```

Comment:
This variable is not used.

Reply:
? The variable is being propagated to the input.nml file in my test runs. see hercules at /home/nbarton/GW/EXPDIR/SFS_C96mx100_S2S/LOGS_COMROOT/1994050100/fcst_mem001_seg0.log

Reply:
Okay, I was looking it in the changes since it was new here, but it is already set up to be used in input.nml.